server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -705,14 +708,22 @@ public void setMailboxesShouldReturnDestroyedWhenChildThenParentMailboxes() {
             .body(ARGUMENTS + ".destroyed", containsInAnyOrder(parentMailboxId, childMailboxId));
     }
 
+    private MailboxId getFakeMailboxId() {
```

Comment:
I would like, but I didn't find an easy way to generate a fake one.
Method renamed.